app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,47 @@
+class ListsController < ApplicationController
+  def index
+    @lists = current_user.lists
+  end
+
+  def create
+    @list = current_user.lists.new
+
+    if @list.update_attributes(list_params)
+      flash[:notice] = 'List successfully created'
+    else
+      flash[:alert] = 'Could not create list'
+    end
+
+    redirect_to action: :index
+  end
+
+  def destroy
+    @list = current_user.lists.find(params[:id])
```

Comment:
suggestion: would it make sense to define a `find_list` method in the user model?